### PR TITLE
add citation information

### PR DESCRIPTION
**Pull request by jakob-r**
#364 

@berndbischl @danielhorn: Can you check if the citing publications are correct?

### CODE REVIEW

**Comment by berndbischl** [comment]
wrt: installation: can we add a sentence that people REALLY PREFERABLY install from cran? 
in my experience too many persons install from GH because it is "easy".

File: README.md [hunk 1]
```diff
@@ -49,5 +57,30 @@ Objective functions are created with package [smoof](https://github.com/jakobbos
...
+  primaryClass = {stat},
+  author = {Bischl, Bernd and Richter, Jakob and Bossek, Jakob and Horn, Daniel and Thomas, Janek and Lang, Michel},
+  date = {2017-03-09},
+}
+```
+
+Some parts of the package were created as part of other publications.
+If you use these parts, please cite the relevant work appropriately:
+
+* Multi-point proposals using multi-objective infill criteria: [MOI-MBO: Multiobjective Infill for Parallel Model-Based Optimization](https://dx.doi.org/10.1007/978-3-319-09584-4_17)
```
Comment (berndbischl) [inline README.md]: i would say:
* Multi-point proposals, including the new multi-objective infill criteria:

(as all other batch proposal code was implemented for this paper, too)

**Comment by jakob-r** [comment]
> in my experience too many persons install from GH because it is "easy".
In which way is the gh install easier than the cran install? I think the motivation behind your experienced "too many gh installs" might be somewhere else :wink:

File: README.md [hunk 2]
```diff
@@ -49,5 +57,30 @@ Objective functions are created with package [smoof](https://github.com/jakobbos
...
+* Multi-objective optimization: [Model-Based Multi-objective Optimization: Taxonomy, Multi-Point Proposal, Toolbox and Benchmark](https://dx.doi.org/10.1007/978-3-319-15934-8_5)
+
```
Comment (berndbischl) [inline README.md]: we could add this here 

First Investigations on Noisy Model-Based Multi-objective Optimization
D Horn, M Dagge, X Sun, B Bischl
International Conference on Evolutionary Multi-Criterion Optimization, 298-313


and also this

http://ieeexplore.ieee.org/document/7850221/?reload=true

Comment (jakob-r) [inline README.md]: But to what parts of mlrMBO do they refer. Noisy Multi Objective is just implemented in a branch so far, isn't it?
And the second one is just an application?

**Comment by jakob-r** [comment]
Taken from an outdated line comment
@berndbischl wrote
> we could add this here
>
>First Investigations on Noisy Model-Based Multi-objective Optimization
D Horn, M Dagge, X Sun, B Bischl
International Conference on Evolutionary Multi-Criterion Optimization, 298-313
>
>and also this
>
>http://ieeexplore.ieee.org/document/7850221/?reload=true

But to what parts of mlrMBO do they refer. Noisy Multi Objective is just implemented in a branch so far, isn't it?
And the second one is just an application?

**Comment by berndbischl** [comment]
can we add the other paper also to the CITATION file? then merge yourself please

**Comment by berndbischl** [comment]
> But to what parts of mlrMBO do they refer. Noisy Multi Objective is just implemented in a branch so far, isn't it?

your are right, @danielhorn should merge this in soon. lets open an issue, not your problem

**Comment by berndbischl** [comment]
> And the second one is just an application?

not really. before that paper, mbo multicrit only worked with GPs. this paper is about the extension with random forests. maybe read it :)

**Comment by jakob-r** [comment]
> can we add the other paper also to the CITATION file? then merge yourself please

Which papers exactly do you want in the CITATION file?

**Comment by jakob-r** [comment]
We can always add more citations later

**Pull request merged**